docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -38,15 +38,19 @@ You can set up CSPM for AWS either by enrolling a single cloud account, or by en
 . Click *Add Cloud Security Posture Management (CSPM)*.
 . Select *AWS*, then either *AWS Organization* to onboard multiple accounts, or *Single Account* to onboard an individual account.
 . Give your integration a name that matches the purpose or team of the AWS account/organization you want to monitor, for example, `dev-aws-account`.
+. (Optional) click **Advanced settings** to deploy the integration using agentless technology (beta). 
+
 
 
 [discrete]
 [[cspm-set-up-cloud-access-section]]
 == Set up cloud account access
-The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access.
+The CSPM integration requires access to AWS's built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access.
 
 For most use cases, the simplest option is to use AWS CloudFormation to automatically provision the necessary resources and permissions in your AWS account. This method, as well as several manual options, are described below.
 
+NOTE: Agentless deployments (beta) support two authentication methods: <<cspm-use-temp-credentials, temporary keys>> and <<cspm-use-keys-directly, direct access keys>>.
```

Comment:
I'm a little wary of adding the beta tag too many times in one page, but I think twice is still OK, and this makes the page consistent too:
```suggestion
NOTE: beta:[] Agentless deployments support two authentication methods: <<cspm-use-temp-credentials, temporary keys>> and <<cspm-use-keys-directly, direct access keys>>.
```